cypress/integration/notes_spec.js

```diff
@@ -1,31 +1,35 @@
 import { GRAPHQL_ENDPOINT } from "../../src/constants";
+import { USERS_QUERY, ADD_NOTE_MUTATION } from "../support/query_constants";
```

Comment:
Probably fine to call the file `queries` (and we know they're constants becuase they're upper case) and safely assume all exports are queries and thus rename `USERS_QUERY` to `USERS`.

Of course this means the mutation should be in another file.